### ISSUE TITLE
Screen reader: play button says "play clip" even though it should say "play episode"

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-web#897]
> this is podverse web with podcasting 2.0, although it requires a bit of fiddling around to know what to click to make it play "the play button is called play clip", all controls are readable https://podverse.fm/episode/ssusVWgnL

**Comment by mitchdowney** [comment]
This should be fixed in v4.6.2
